01/part1.py

```diff
@@ -6,10 +6,8 @@
 totals = []
 
 with open("01_p1.txt", "r") as f:
-    line_number = 0
```

Comment:
Good call here. I'd originally left it in because I can never anticipate what "extra details" part 2 will ask for, so something I throw extra bits in just in case I inadvertently catch what they're looking for without having to go back and add anything extra 😀 But yes in this case it's just fluff. Thanks!